cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1425,6 +1425,22 @@
     </description>
   </property>
 
+  <property>
+    <name>log.buffer.server.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      Log buffer HTTP service bind address
+    </description>
+  </property>
+
+  <property>
+    <name>log.buffer.server.bind.port</name>
```

Comment:
This is because we started adding bind host and port default entries as per https://github.com/cdapio/cdap/pull/11001/files#diff-d2f273e30d51501da90e3f8cea5929e0R2132

Reply:
In K8s, it cannot be random port if there are more than one instance behind a Service endpoint.